jni/agc.cpp

```diff
@@ -11,10 +11,11 @@
 /*-------------------[      Project Include Files      ]-------------------*/
 #include "filter_audio/agc/include/gain_control.h"
 /*-------------------[      Macros/Constants/Types     ]-------------------*/
+#define MIC_MAX      255   // maximum virtual mic level
+#define MIC_TARGET   180   // -3dBFS
```

Comment:
I noticed that this is marked as a negative amplitude here but positive in the component.

Reply:
the the virtual mic levels are a bit different than the agc's target-level-dbfs - the virtual mic is like a slider the user could use to control the input levels at runtime
but yeah, webrtc inverts the sign on agc-compression-gain-db internally